wordpress-mobile/WordPress-iOS
# Quick Start: Display modal after login if default section is Home

Fixes #18566 

## Description
Quick Start modal was only being displayed after login if the assigned default tab is Menu. This PR adds the same behaviour to user with default tab Home.

https://user-images.githubusercontent.com/25306722/170143008-e2b2632d-eb85-4891-bfe2-f967b77ec0f5.mp4

## Testing Instructions

1. Replace the implementation of BlogDashboardAB.calculateVariant() with return .treatment
2. Install and run the app
3. Log in then and choose "show me around"
4. Make sure that the quick start modal is presented

## Regression Notes
1. Potential unintended areas of impact
Removing quick start cards and updating quick start cards when "Initial Screen" changes

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually 

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
